ta_services/keymaster/private_include/user_ta_header_defines.h

```diff
@@ -0,0 +1,22 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/* Copyright (c) 2018, Linaro Limited */
+
+#ifndef __USER_TA_HEADER_DEFINES_H
+#define __USER_TA_HEADER_DEFINES_H
+
+#include <keymaster_ta.h>
+
+#define TA_UUID				TA_KEYMASTER_UUID
+
+#define TA_FLAGS			(TA_FLAG_SINGLE_INSTANCE | \
```

Comment:
is there any requirement that it should be `TA_FLAG_SINGLE_INSTANCE`?

Reply:
Let's turn that around, is there any requirement that it mustn't be single-instance? None as far as I know, single-instance is more safe though.